app/assets/javascripts/application/suggest_user.js

```diff
@@ -0,0 +1,10 @@
+function suggestUser(){
+  $.get( '/search/suggest_leader',{query:   $("[name='group_leader_name']")[0].value},function( data ) {
+      $( ".result" ).html(  );
+      $("#user_suggestions").children().remove()
+      $("#user_suggestions").append(data)
+    });
+}
+function selectUser(user_id){
+    $("#selected_user_id")[0].value=user_id
```

Comment:
It would be better if we used Pure JavaScript instead of JQuery.

Reply:
Okay, I will try to rewrite it in pure JS!